packages/telemetry/src/slog-to-otel.js

```diff
@@ -501,60 +585,156 @@ export const makeSlogToOtelKit = (tracer, overrideAttrs = {}) => {
             },
             options,
           );
+        const syscallType = ksc[0];
+        const name = `syscall-${syscallType}`;
         // TODO: get type from packages/SwingSet/src/kernel/kernelSyscall.js
-        switch (ksc[0]) {
+        switch (syscallType) {
           case 'send': {
             const [
               _tag,
               target,
               { methargs, method = extractMethod(methargs), result },
             ] = ksc;
-            const name = `E(${target}).${method}`;
+            const details = `E(${target}).${method}`;
```

Comment:
Why not have the same prefixed format as the other syscalls `s-${vatID}...: `. ?

Note: I think the prefix should be computed by some common logic, maybe in `makeSyscallSpan`? Especially since we'll likely have to defer the span creation until deliver-result.